.github/scripts/promote_branch.sh

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env bash
+
+# This script promotes branches in the release-service-catalog repository.
+#
+# The script promotes the development content into the staging branch, or the staging
+# content into the production branch. It starts by performing the following checks, then
+# it performs a git push. There is no pull request.
+#
+# Checks:
+#   - If there is content in the staging branch that is not yet in the production branch, the
+#     script will not git push to add more content to the staging branch. This can be overridden with
+#     --force-to-staging true
+#   - If promoting to production and the content has not been in the staging branch for at least 7 days,
+#     the script will exit without doing a push. Content is expected to sit in staging for at least a week
+#     to provide sufficient testing time. This can be overridden with --override true
+#
+# Prerequisities:
+#   - An environment variable GITHUB_TOKEN is defined that provides access to the user's account. See
+#     https://github.com/redhat-appstudio/release-service-utils/blob/main/ci/promote-overlay/README.md#setup for help.
+#   - curl, git and jq installed.
+
+set -e
+
+# GitHub repository details
+ORG="redhat-appstudio"
+REPO="release-service-catalog"
+
+OPTIONS=$(getopt --long "target-branch:,force-to-staging:,override:,debug,help" -o "tgt:,h" -- "$@")
+eval set -- "$OPTIONS"
+while true; do
+    case "$1" in
+        -tgt|--target-branch)
+            TARGET_BRANCH="$2"
+            shift 2
+            ;;
+        --force-to-staging)
+            FORCE_TO_STAGING="$2"
+            shift
```

Comment:
Should be `shift 2`, right? Same with `--override`.

Reply:
Nice catch, fixed